contracts/VRTVault/VRTVault.sol

```diff
@@ -63,13 +72,15 @@ contract VRTVault is VRTVaultStorage {
         _;
     }
 
-    function pause() public onlyAdmin {
+    function pause() external {
```

Comment:
External and public functions should have natspec

Reply:
These functions should also come after the modifiers

Reply:
Fixed [here](https://github.com/VenusProtocol/venus-protocol/pull/236/commits/0b23a7600272fd7b1b932845d5c5b914a567d909)